src/main/fc/fc_msp.c

```diff
@@ -586,6 +596,256 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFn
         sbufWriteData(dst, shortGitRevision, GIT_SHORT_REVISION_LENGTH);
         break;
 
+
+    case MSP_REBOOT:
+        if (mspPostProcessFn) {
+            *mspPostProcessFn = mspRebootFn;
+        }
+        break;
+
+    case MSP_ANALOG:
+        sbufWriteU8(dst, (uint8_t)constrain(getBatteryVoltage(), 0, 255));
+        sbufWriteU16(dst, (uint16_t)constrain(getMAhDrawn(), 0, 0xFFFF)); // milliamp hours drawn from battery
+        sbufWriteU16(dst, 0); // rssi
+        sbufWriteU16(dst, (int16_t)constrain(getAmperage(), -0x8000, 0x7FFF)); // send current in 0.01 A steps, range is -320A to 320A
+        break;
+
```

Comment:
Whitespace - let's be consistent and have a single blank line at the end of each `case`.